lib/inquirer.js

```diff
@@ -23,8 +23,8 @@ inquirer.ui = {
  * Create a new self-contained prompt module.
  */
 inquirer.createPromptModule = function () {
-  var promptModule = function (questions, allDone) {
-    var ui = new inquirer.ui.Prompt(promptModule.prompts);
+  var promptModule = function (questions, allDone, opt) {
```

Comment:
I think `inquirer.createPromptModule()` should take input/output options. `inquirer.prompt` should stay as is.